src/main/resources/idprovider/idprovider.ts

```diff
@@ -1,16 +1,21 @@
-const mustacheLib = require('/lib/mustache');
-const portalLib = require('/lib/xp/portal');
-const authLib = require('/lib/xp/auth');
-const adminCreationLib = require('/lib/admin-creation');
-const adminLib = require('/lib/xp/admin');
-const autoLoginLib = require('/lib/autologin');
-const configLib = require('/lib/config');
-const staticLib = require('/lib/enonic/static');
+import type {Request} from '@item-enonic-types/global/controller';
```

Comment:
Are we using `@item-enonic-types/global` implicitly, via transitive dependency from other libs? We should use our own, from https://github.com/enonic/xp/issues/9742